app/views/aleph/full_item_status.html.erb

```diff
@@ -1,69 +1,82 @@
+      <li class="result-local-location">
+        <div class="location-wrap">
+          <span class="location">
+            <span class="library"><%= s[:library] %></span>
+            <span class="collection"><%= s[:collection] %>:</span>
+            <span class="callno"><%= s[:call_number] %></span>
+            <span class="desc">
+              <% if s[:description].present? %> <%= s[:description] %> <% end %>
+            </span>
+            <span class="sr"><%= s[:label] %></span>
+          </span>
+          <span class="location-actions">
+            <a href="<%= map_link(s[:library]) %>"
+               class="btn button-subtle map-link">
+               <i class="fa fa-map-marker" aria-hidden="true"></i>
+               <span class="sr">View <%= s[:library] %> location map</span>
+            </a>
+          </span>
+        </div>
+        <div class="availability-wrap">
 
-            <a class="btn button-secondary button-small" href="">Request Scan</a>
+          <% if s[:available?] %>
+            <span class="availability-status">
+              <i class="fa fa-check" aria-hidden="true"></i>
+              Available
+            </span>
+            <span class="availability-actions">
 
-          <% end %>
+              <% if archives?(s[:library]) %>
+
+                <a class="btn button-secondary button-small" href="https://libraries.mit.edu/archives/">Contact Us</a>
+
+              <% elsif reserve?(s[:collection]) %>
+
+                <a class="btn button-secondary button-small" href="">Place Hold</a>
+
+              <% elsif media?(s[:call_number]) %>
+
+                <a class="btn button-secondary button-small" href="">Place Hold</a>
 
-        </span>
+              <% else %>
 
-      <% else %>
-        <span class="availability-status">
-          <i class="fa fa-times" aria-hidden="true"></i>
-          Not available at MIT
-        </span>
-        <span class="availability-actions">
-          <a class="btn button-subtle button-small" href="">Recall (7+ days)</a>
-          <a class="btn button-secondary button-small" href="">Get it with ILL (3-4 days)</a>
-        </span>
+                <a class="btn button-secondary button-small" href="">Place Hold</a>
+
+                <a class="btn button-secondary button-small" href="">Request Scan</a>
+
+              <% end %>
+
+            </span>
+
+          <% else %>
+            <span class="availability-status">
+              <i class="fa fa-times" aria-hidden="true"></i>
+              Not available at MIT
+            </span>
+            <span class="availability-actions">
+              <a class="btn button-subtle button-small" href="">Recall (7+ days)</a>
+              <a class="btn button-secondary button-small" href="">Get it with ILL (3-4 days)</a>
+            </span>
+
+          <% end %>
 
-      <% end %>
+        </div>
+      </li>
+    <% end %>
+    </ul>
 
-    </div>
-  </li>
+<% if @status.count > 5 %>
```

Comment:
Either this if block, or the one at the top of the page, or both, should have a comment referencing the other, so that we-in-the-future don't accidentally edit them asymmetrically and break stuff.

Reply:
Yeah, that's a mess. Let me see if I can make it less messy (and if not I'll take the note path)

Reply:
@frrrances is going to refactor the toggler js to remove the necessity of adding the wrappers in app. If that work is done soonish I'll undo most of the work in this branch and add just the stuff necessary (removing the restriction on serials availability displays)

Reply:
👍